paper-ieee-trans.tex

```diff
@@ -159,9 +162,9 @@ \section{Evolution of research on software reliability in EC--projects}
 
 The EC projects considered in this paper are listed chronologically in Table~\ref{tab:eup}.
 Each project paved the way for the following ones, according to a logic evolution in which
-the role of the quality and reliability of the software have
+the role of the quality and reliability of the software has
 consistently increased. As shown in Table~\ref{tab:feat}, the most recent projects are more
-committed to and aware of software reliability practices.
+committed to, and aware, of software reliability practices.
```

Comment:
i would remove the commas or put the "of" inside

Reply:
"of" is now inside the commas
if OK approve, soon will have the refs fixed